src/app/zap-templates/zcl/data-model/silabs/ha.xml

```diff
@@ -53,11 +53,61 @@ limitations under the License.
     <define>FAN_CONTROL_CLUSTER</define>
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
-    <attribute side="server" code="0x0000" define="FAN_CONTROL_FAN_MODE" type="ENUM8" min="0x00" max="0x06" writable="true" default="0x05" optional="false">fan mode</attribute>
-    <!-- FAN_MODE -->
-    <attribute side="server" code="0x0001" define="FAN_CONTROL_FAN_MODE_SEQUENCE" type="ENUM8" min="0x00" max="0x04" writable="true" default="0x02" optional="false">fan mode sequence</attribute>
-    <!-- FAN_MODE_SEQUENCE -->
+    <attribute side="server" code="0x0000" define="FAN_MODE" type="ENUM8" min="0x00" max="0x06" writable="true" default="0x00" optional="false">fan mode</attribute>
```

Comment:
Why is the type here not FanModeType or FanModeEnum (whatever the enum should actually be called)?  Similar for FanModeSequence....